packages/api/types.ts

```diff
@@ -2319,7 +2339,30 @@ type ChangeEmailResponse = {
     msg_type: 'change_email';
     req_id?: number;
 };
-
+/**
+ * Get list of platform and their server status
+ */
+type TradingPlatformStatusRequest = {
+    /**
+     * Must be 1
+     */
+    trading_platform_status: 1;
```

Comment:
why 1?

Reply:
that's the only payload required to send to BE to get response